ush/config_defaults.sh

```diff
@@ -480,21 +493,13 @@ RESTART_INTERVAL="3,6"
 # cycle definition for "boundary_long" group
 # This group runs: get_extrn_lbcs_long,make_lbcs
 #
-# PREP_COLDSTART_CYCLEDEF:
-# cycle definition for "prep_coldstart" group
-# This group runs: prep_coldstart
-#
-# PREP_WARMSTART_CYCLEDEF:
-# cycle definition for "prep_warmstart" group
-# This group runs: prep_warmstart
+# SPINUP_CYCLEDEF:
+# cycle definition for spin-up cycle group
+# This group runs: anal_gsi_input_spinup and data process, run_fcst_spinup, run_post_spinup
```

Comment:
I failed to mention this before, but this is a really nice consolidation of cycledefs! :)